0.8.0 (#94)
uploadcare/uploadcare-swift: README.md

(task as? UploadTaskResumable)?.resume()
```

> It is possible to perform uploads in background but since it's implementation is platform-specific the lib doesn't provide default implementation. Anyway you can fine an example for iOS in our Demo app. See [FilesListStore.swift](https://github.com/uploadcare/uploadcare-swift/blob/1e6341edcdcb887589a4e798b746c525c9023b4e/Demo/Demo/Modules/FilesListStore.swift).

Reviewer: ```suggestion
It is possible to perform uploads in background. But implementation is a platform-specific. This lib doesn't provide default implementation. You can find an example for the iOS in our Demo app. See [FilesListStore.swift](https://github.com/uploadcare/uploadcare-swift/blob/1e6341edcdcb887589a4e798b746c525c9023b4e/Demo/Demo/Modules/FilesListStore.swift).
```
